public/externalLibs/env_visualizer/visualizer.js

```diff
@@ -13,111 +15,10 @@
     height: 1000,
     container: container
   });
-  const frameFontSetting = '14px Roboto Mono, Courier New';
+  const frameFontSetting = '14px RobotGo Mono, Courier New';
```

Comment:
What is robotgo font?

Reply:
Hmm the `RobotGo Mono` doesn't exist. Unintended typo, @podocarp?

Reply:
That's odd. Don't recall trying to change that. I'll fix it.